contracts/deploy/031_staking_add_rescue.js

```diff
@@ -0,0 +1,42 @@
+const { deploymentWithProposal } = require("../utils/deploy");
+
+const TRANSFER_AGENT = "0x522731a061e896B5Db9dDff9234fB5461A533710";
```

Comment:
Who dis 🤣

Reply:
Checked with the authority. Let's use 0x449E0B5564e0d141b3bc3829E74fFA0Ea8C08ad5

Reply:
Used it